Sprint-3/revise/implement/get-ordinal-number.test.js

```diff
@@ -1,4 +1,45 @@
 // In this week's prep, we started implementing getOrdinalNumber
+//
+//const { default: test } = require("node:test");
+
+//const { rmdir } = require("fs")
 
 // continue testing and implementing getOrdinalNumber for additional cases
 // Write your tests using Jest - remember to run your tests often for continual feedback
+function getOrdinalNumber(num) {
+  let lastDigit = num % 10;
+  if (lastDigit == 1) return num + "st";
+  if (lastDigit == 2) return num + "nd";
+  if (lastDigit == 3) return num + "rd";
+  if (lastDigit >= 4 || lastDigit == 0) return num + "th";
```

Comment:
Logically, if `lastDigit` is not 1, 2, or 3, then it must be 0, 4-9. So the check at line 14 is optional.

But your function needs to check for some other cases because `getOrdinalNumber(11)` should return "11th" instead of "11st". You can use Google/ChatGPT to find out the rules to convert a number to an ordinal number.